examples/common/chip-app-server/Server.cpp

```diff
@@ -47,13 +53,75 @@ using namespace ::chip::DeviceLayer;
 DemoSessionManager sessions;
 
 namespace chip {
+
 SecureSessionMgrBase & SessionManager()
 {
     return sessions;
 }
+
 } // namespace chip
 
 namespace {
+
+void HandleBLEConnectionClosed(chip::Ble::BLEEndPoint * endPoint, BLE_ERROR err)
+{
+    ChipLogProgress(AppServer, "BLE Connection closed");
+}
+
+void HandleBLEMessageReceived(chip::Ble::BLEEndPoint * endPoint, chip::System::PacketBuffer * buffer)
+{
+#if CHIP_ENABLE_OPENTHREAD
+    uint16_t bufferLen = buffer->DataLength();
+    uint8_t * data     = buffer->Start();
+    chip::DeviceLayer::Internal::DeviceNetworkInfo networkInfo;
+    ChipLogProgress(AppServer, "Receive message size %u", bufferLen);
```

Comment:
How about also log the peer's BLE address?

```suggestion
    ChipLogProgress(AppServer, "Receive BLE message. size=%u peer=%s", bufferLen, XXX);
```